CCS2.0/CCS2.0/Controllers/HomeController.cs

```diff
@@ -64,19 +64,20 @@ public IActionResult Index(string sBase64String, ImageModel model)
 
 
 
-    public IActionResult ViewPost(int ID)
+    public IActionResult ViewPost(int ID, int page = 1)
         {
             List<ImageModel> Match = new List<ImageModel>();
             List<ImageUpload.Models.CommentModel> Com = new List<ImageUpload.Models.CommentModel>();
             List<ImageUpload.Models.AddingTagModel> Tag = new List<ImageUpload.Models.AddingTagModel>();
 
             var match = GetPhotoId(ID);
 
-
-            var com = GetCommentId(ID);
+            var com = GetCommentId(ID, page);
 
             var tag = getTagId(ID);
 
+            var Pages = (int)Math.Ceiling((decimal)GetPages(ID)/5);
```

Comment:
Divide by 5? That's arbitrary. I guess for this view it's OK. Maybe make it a default param? Or controlled in the UI.